draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -3926,23 +3930,29 @@ Content-Range: exampleunit 11.2-14.3/25
    the response might differ.
 </t>
 <t>
-   A user agent &MUST-NOT; automatically retry a request with a non-idempotent
-   method unless it has some means to know that the request semantics are
-   actually idempotent, regardless of the method, or some means to detect that
-   the original request was never applied. For example, a user agent that
-   knows (through design or configuration) that a POST request to a given
-   resource is safe can repeat that request automatically.
-   Likewise, a user agent designed specifically to operate on a version
-   control repository might be able to recover from partial failure conditions
-   by checking the target resource revision(s) after a failed connection,
-   reverting or fixing any changes that were partially applied, and then
-   automatically retrying the requests that failed.
+   A client &SHOULD-NOT; automatically retry a request with a
```

Comment:
This changes "user-agent" to "client", but a subsequent use of "user-agent" stays. Intentional?

Reply:
Yes. This allows non-proxy intermediaries to do this, which they already do.